Libraries/test/Amazon.Lambda.RuntimeSupport.Tests/Amazon.Lambda.RuntimeSupport.IntegrationTests/BaseCustomRuntimeTest.cs

```diff
@@ -217,7 +224,8 @@ protected async Task CreateFunctionAsync(IAmazonLambda lambdaClient, string buck
                 },
                 Handler = "PingAsync",
                 MemorySize = 512,
-                Runtime = Runtime.Provided,
+                Timeout = 30,
```

Comment:
I needed to increase the timeout to give enough to to prove I was not getting a deadlock